### DISCUSSION TITLE
draft-ietf-ipsecme-ikev2-multiple-ke

### DISCUSSION THREAD

**Discussion by keskinadem** [strongswan/strongswan#1538] [Q&A] [answered]
ikev2-multiple-key-exchange is in Internet-Draft version. Info that is written in strongswan about Multiple Key Exchange is just "Added IANA numbers assigned to draft-ietf-ipsecme-ikev2-multiple-ke" Here is link: https://download.strongswan.org/CHANGES6dr.txt What is "Multiple Key Exchanges in IKEv2" internet-draft version? and Can I use strongswan-6.0beta2 for Multiple Key Exchange?

**Comment by tobiasbrunner** [comment, marked as answer]
The beta release implements the latest version of the draft.
